src/extensions/default/Brackets-InteractiveLinter/main.js

```diff
@@ -21,11 +21,7 @@ define(function (require, exports, module) {
         linterManager      = require("linterManager"),
         pluginManager      = require("pluginManager");
         
-	function addEnabledPref() {
-        preferences.definePreference("enabled", "boolean", true);
-    }
-
-    require("lintIndicator");
+	require("lintIndicator");
```

Comment:
Look at this.  Surely you can see that it's not aligned with what's before/after it?  Why is your indentation so wrong everywhere?  Is your editor setup incorrectly?